### PR TITLE
Add HTTP port to trace ingestion

### PR DESCRIPTION
**Pull request by petewall**

### CODE REVIEW

File: charts/k8s-monitoring/templates/_helpers.tpl [hunk 1]
```diff
@@ -1,22 +1,22 @@
 {{/* This template checks that the port defined in .Values.traces.receiver.port is in the targetPort list on .grafana-agent */}}
```
Comment (skl) [inline charts/k8s-monitoring/templates/_helpers.tpl:1]:
```suggestion
{{/* This template checks that the port defined in .Values.traces.receiver.*.port is in the targetPort list on .grafana-agent */}}
```

File: charts/k8s-monitoring/templates/agent_config/_traces.river.txt [hunk 2]
```diff
@@ -1,16 +1,26 @@
 {{ define "agent.config.traces" }}
-{{- include "checkForTracePort" . -}}
-{{- with .Values.traces }}
 // Traces
 otelcol.receiver.otlp "trace_receiver" {
+{{- if .Values.traces.receiver.grpc.enabled }}
```
Comment (skl) [inline charts/k8s-monitoring/templates/agent_config/_traces.river.txt:4]: The older .receiver.port was kept for compatibility but looks like it was dropped from the template iiuc - do you need another conditional to check for that?

**Review by skl — APPROVED** [review]
lgtm - just a compatibility comment

File: charts/k8s-monitoring/templates/agent_config/_traces.river.txt (hunk 2 above)
Comment (petewall) [reply charts/k8s-monitoring/templates/agent_config/_traces.river.txt:4]: Yes, thank you!

**Pull request merged**